browser/src/plugin.ts

```diff
@@ -24,20 +70,13 @@ class ExtismPluginCall {
       extism_load_u8(n: bigint): number {
         return plugin.allocator.memory[Number(n)];
       },
-      extism_load_u32(n: bigint): number {
```

Comment:
Should we leave these but just throw an exception? what happens if you use a plugin that expects them to be there?

Reply:
there shouldnt be any plugins that expect these -- weve also removed them from all other sdks so this is consistent with the rest of the world

Reply:
yeah, these functions were removed but also have never been used in any of the PDKs, so it's unlikely to find a plugin that uses them.

Reply:
IIRC, the older plugins expect the whole interface to be satisfied even though they don't use them

Reply:
i think that's only true if the function is actually called somewhere